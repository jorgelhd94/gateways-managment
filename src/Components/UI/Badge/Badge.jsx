import React from 'react';

const Badge = ({children}) => {
  return (
    <>
      <span className="px-1 bg-blue-500 text-white rounded-full">{children}</span>
    </>
  );
};


export default Badge;
