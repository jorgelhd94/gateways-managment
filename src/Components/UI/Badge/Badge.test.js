import React from 'react';
import renderer from 'react-test-renderer';
import Badge from './Badge';

test('Badge with primary style', () => {
  const component = renderer.create(<Badge type="primary">1</Badge>);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
