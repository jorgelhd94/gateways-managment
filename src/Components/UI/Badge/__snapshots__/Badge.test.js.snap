// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge with primary style 1`] = `
<span
  className="px-2 py-1 text-white rounded-full bg-blue-500"
>
  1
</span>
`;
